From d282a34dfc3667250a8878af245906d0053b0b33 Mon Sep 17 00:00:00 2001
From: Peter Delevoryas <pdel@fb.com>
Date: Sun, 14 Aug 2022 23:43:55 -0700
Subject: [PATCH] bletchley: Increase RAM from 512MB to 2GB

Quanta is experimenting with switching Bletchley from 512MB of DRAM to
2GB. The device tree has been updated, so this updates QEMU.

You may or may not need the kernel arg `vmalloc=384M` to boot
successfully with this, because the default kernel memory allocation
seems to decrease and cause a kernel panic.

Signed-off-by: Peter Delevoryas <pdel@fb.com>
---
 hw/arm/aspeed.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/hw/arm/aspeed.c b/hw/arm/aspeed.c
index 7b44bc2..2fc22f7 100644
--- a/hw/arm/aspeed.c
+++ b/hw/arm/aspeed.c
@@ -1442,7 +1442,7 @@ static void aspeed_machine_bletchley_class_init(ObjectClass *oc, void *data)
     amc->num_cs    = 2;
     amc->macs_mask = ASPEED_MAC2_ON;
     amc->i2c_init  = bletchley_bmc_i2c_init;
-    mc->default_ram_size = 512 * MiB;
+    mc->default_ram_size = 2 * GiB;
     mc->default_cpus = mc->min_cpus = mc->max_cpus =
         aspeed_soc_num_cpus(amc->soc_name);
 }
-- 
2.37.2

